### PR TITLE
Electrum Streaming

### PR DESCRIPTION
**Pull request by evanlinjin**

### CODE REVIEW

File: frostsnap_coordinator/src/bitcoin/chain_sync.rs [hunk 1]
```diff
@@ -107,155 +137,208 @@ pub const fn default_electrum_server(network: bitcoin::Network) -> &'static str
...
+        url: String,
+        wallet: Arc<std::sync::Mutex<FrostsnapWallet>>,
+        mut update_action: impl FnMut(MasterAppkey, Vec<crate::bitcoin::wallet::Transaction>)
+            + Send
+            + 'static,
+    ) {
+        let (emitter, cmd_sender, mut update_recv) = {
+            let wallet = wallet.lock().unwrap();
+            // let lookahead = wallet.lookahead();
+            let lookahead = 20;
```
Comment (evanlinjin) [inline frostsnap_coordinator/src/bitcoin/chain_sync.rs:169]: Woops. Should use `wallet.lookahead()`.

File: frostsnapp/flutter_jank_metrics_01.json
Comment (evanlinjin) [inline frostsnapp/flutter_jank_metrics_01.json:1]: What is this?

File: frostsnapp/lib/wallet.dart [hunk 2]
```diff
@@ -835,49 +788,49 @@ String formatSatoshi(int satoshis) {
...
+//        //walletCtx.startFullSync();
+//      }
+//    });
+//  }
+//
+//  @override
+//  Widget build(BuildContext context) {
+//    return widget.child;
+//  }
+//}
```
Comment (evanlinjin) [inline frostsnapp/lib/wallet.dart:836]: Rm.

**Review by evanlinjin — COMMENTED** [review]
ACK 33205efec43c3d2f97bbbde13b991a2667698e5b

**Review by LLFourn — APPROVED** [review]

**Pull request merged**